### PR TITLE
Allow copy left plugins in the Marketplace

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
As discussed on Community there is no need to prohibit copy left plugins in the Marketplace.

#### Ticket Link
https://community.mattermost.com/private-core/pl/mk66rxs8etbffd8gc9bg9mfone

### CODE REVIEW

File: site/content/extend/plugins/community-plugin-marketplace.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Every community plugin must fulfill the following checklist to be added to the M
 
 **Product Requirements (Checked by a Product Manager)**
 
-1. The plugin is published under an Apache v2 compatible license (e.g. no GPL, APGL). A list of compatible licenses can be found [here](https://apache.org/legal/resolved.html#category-a).
+1. The plugin is published under an [Open Source license](https://opensource.org/osd-annotated).
```
[the remark this answers is not in the document]
Comment (jasonblais) [reply site/content/extend/plugins/community-plugin-marketplace.md:16]: Proposed update as this link is used by Atlassian and GitLab for their open source programs
 - https://opensource.org/licenses/alphabetical
 - https://opensource.org/licenses/category

**Review by jasonblais — CHANGES_REQUESTED** [review]
If we do get a plugin submission with a copy left license, we'll have a legal review at that point to confirm there are no concerns, correct?

**Comment by hanzei** [comment]
> If we do get a plugin submission with a copy left license, we'll have a legal review at that point to confirm there are no concerns, correct?

Yes, we should do that.

**Review by jasonblais — APPROVED** [review]
Is the legal review included in a process somewhere? Otherwise it'll be forgotten if a copy-left license is submitted a year from now, for instance?

**Comment by hanzei** [comment]
The process owner i.e. me checks what license a submitted plugins has. Confirming that the license doesn't contain parts that Mattermost can't fulfill, is part of that check. 2/5 that a todo to do legal review shouldn't be part of this document as the audience are community members.

**Review by jespino — APPROVED** [review]
LGTM

**Pull request merged**